generator/model.go

```diff
@@ -1733,6 +1753,53 @@ func goName(sch *spec.Schema, orig string) string {
 	return orig
 }
 
+// buildMapOfNullable equalizes the nullablity status for aliased and anonymous maps of simple things,
+// with the nullability of its innermost element.
+//
+// NOTE: at the moment, we decide to align the type of the outer element (map) to the type of the inner element
+// The opposite could be done and result in non nullable primitive elements. If we do so, the validation
+// code needs to be adapted by removing IsZero() and Required() calls.
+func (sg *schemaGenContext) buildMapOfNullable(sch *GenSchema) {
```

Comment:
cyclomatic complexity 22 of func `(*schemaGenContext).buildMapOfNullable` is high (> 10)